src/frontend/src/utils/statusList/statuses/incompleteStatus.ts

```diff
@@ -24,27 +25,34 @@ export const incompleteStatus = (pia: IPiaForm | null) => {
         showSubmitButton: true,
         changeStatus: [
           {
-            status: 'EDIT_IN_PROGRESS',
+            status: 'EDIT_IN_PROGRESS', // change to EDIT_IN_PROGRESS
             modal: checkReviewStatus(pia)
               ? resetReviewEditInPRogressModal
               : defaultEditInProgressModal,
           },
           {
-            status: 'MPO_REVIEW',
-            modal: defaultMPOReviewModal,
+            status: 'MPO_REVIEW', // change to MPO_REVIEW
+            modal:
+              from === 'submit' ? submitPiaIntakeModal : defaultMPOReviewModal,
```

Comment:
should not a 'from' be a previous status? why should 'from' be a submit?

Reply:
I dont think "from" is best term to use either. I think we should just set a `modal` and `submitModal` property

Reply:
i think 'previous state' might be better than 'from'. 'Submit' is an action from one state (previous status) to the next state (next status). The 'Submit' button could change tomorrow to 'next steps' or have some-other functionality. What should be consistent is tracking the states. 
The state table should avoid logic of what triggered (button, dropdown) the state change but keep track of the previous state and next state and compute logic accordingly.

My 2 cents.